### ISSUE TITLE
Misleading docstring for argument `palette` in iqplot.ecdf().

### ISSUE THREAD

**Issue by tomroesch** [justinbois/iqplot#12]
Docstring says that `palette` can take a string as input. 

<img width="494" alt="image" src="https://user-images.githubusercontent.com/55992574/133511438-816ebfae-81a6-40d2-8cf3-56f09c431737.png">

However, it gives an error message.

```
import iqplot
import pandas as pd
import numpy as np
import bokeh.io

bokeh.io.output_notebook()

df = pd.DataFrame(data=np.random.randn(1000), columns=['x'])

bokeh.io.show(
    iqplot.ecdf(
        data=df,
        q='x',
        palette='orange'
    )
)
```

<img width="935" alt="image" src="https://user-images.githubusercontent.com/55992574/133511214-4b4b752e-15fd-444d-b89e-36bee38b2a1a.png">

**Comment by atisor73** [comment]
@tomroesch, could you do: 
```
if type(palette) == str:
    palette = [palette]
```
instead of changing the docstring?

**Comment by tomroesch** [comment]
I added a pull request that does exactly this.

**Comment by atisor73** [comment]
ah that's great! sorry I didn't see it

**Comment by justinbois** [comment]
Fixed with #13
